recipe/run_test.py

```diff
@@ -1,28 +1,37 @@
+import subprocess
 from dateutil.relativedelta import relativedelta
 from dateutil.easter import easter
 from dateutil.rrule import rrule, YEARLY, FR
 from dateutil.parser import parse
 from dateutil.tz import tzlocal
+from dateutil import zoneinfo, utils
+from importlib_metadata import version
+
+
+# Test pip check
+subprocess.run(["pip", "check"])
+
+print(version("python-dateutil"))
```

Comment:
so this is expected to return "2.8.3+snowflake1 " ? 
can we add/replace with assert  instead ?

Reply:
Thanks. Fixed it.